libraries/AP_Motors/AP_MotorsHeli_Swash.h

```diff
@@ -49,6 +49,11 @@ class AP_MotorsHeli_Swash {
     void set_servo3_pos(int16_t servo_pos) { _servo3_pos = servo_pos; }
     void set_servo4_pos(int16_t servo_pos) { _servo4_pos = servo_pos; }
 
+    // set_linear_servo_out - sets swashplate servo output to be linear
+    void set_linear_servo_out(int8_t linear_servo) { _make_servo_linear = linear_servo; }
+
+    //linearize mechanical output of swashplate servo
+    float get_linear_servo_output(float input);
```

Comment:
might be good to const the get_linear_servo_output function.  while there maybe add the space after the "//" in the comments above.